Lesson5

Shepico opened the pull request:
Вроде все ок.

Делал все в одном классе

Shepico committed:
Создал 2 метода. Считает оба.
Во втором методе 2 потока.

Shepico committed:
Удалил лишнее и не используемое

Ivan-Igorevich commented on src/lesson5/L5_MainClass.java:
```diff
@@ -0,0 +1,99 @@
+        Thread t2 = new Thread(new Runnable() {
+            @Override
+            public void run() {
+                int i=5000000;
```
костыль))) а если поделить надо будет не на два потока, а на три, например, можно написать какой то универсальный раннебл, который смог бы работать в такой ситуации?

Shepico committed:
Замечания
костыль))) а если поделить надо будет не на два потока, а на три, например, можно написать какой то универсальный раннебл, который смог бы работать в такой ситуации?
========
Устранил, вместо 500, теперь на сколько делим

Shepico committed:
добавил еще ожидание первым потоком второго
